src/analysis/raster/qgsrastercalcnode.h

```diff
@@ -125,6 +130,13 @@ class ANALYSIS_EXPORT QgsRasterCalcNode
 
     static QgsRasterCalcNode *parseRasterCalcString( const QString &str, QString &parserErrorMsg ) SIP_FACTORY;
 
+    /**
+     * Calculates result of raster calculation when tFunct type is used
+     * \since QGIS 3.22
+     */
+    QgsRasterMatrix evaluation( const QVector<QgsRasterMatrix *> &matrixVector, QgsRasterMatrix &result ) const;
```

Comment:
1. let's rename the function to something more descriptive - e.g. `evaluateFunction` - to make it clear this is used for functions
2. make the function private - there's no need to have it in public api